### PR TITLE
Document browser support

### LINKED ISSUE #155: Think about and document browser support
We should probably explicitly think about and then eventually stipulate and document which browsers we want to support.

See also the `browserlist` setting in `package.json`, which is still `crreate-react-app`-s default, if I'm not mistaken. Is this default sensible (for our use cases)?

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
Closes #155 

The configuration in `package.json` is just relevant for Babel and does not mean we support all browsers included there. Currently the queries include IE11 as it still has roughly 1.5% market share. And as adding `"not ie 11"` does not shrink the production JS files, there is no point in doing so IMO.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,3 +75,12 @@ If you prefer to run a local **development** server directly, you can use instea
 ```sh
 % npm run start
 ```
+
+Supported Browsers
+------------------
+
+This application should run on the latest version of the following browsers:
```
Comment (LukasKalbertodt) [inline README.md]: Or should I write "This application *runs* on the..."? Depends on how confident we are in our QA :grin:

Comment (lkiesow) [reply README.md]: Indeed, this sounds a bit like we aren't sure.
While, of course, we aren't, maybe just use something like:

> The following browsers are officially supported:

Comment (luniki) [reply README.md]: According to https://browserl.ist/?q=%3E1%25%2C+not+dead%2C+not+op_mini+all%2C+last+2+chrome+version%2C+last+2+firefox+version%2C+last+2+safari+version the following browsers are now selected:

image

Comment (LukasKalbertodt) [reply README.md]: @luniki Which notably excludes mobile Firefox. Thanks for checking this, I will add that browser explicitly.

**Comment by LukasKalbertodt** [comment]
@luniki @lkiesow Just as an example, in PRs like these, I usually add the review suggestions as fixup to the existing commits and force push. Especially if the fixes are straight forward. I also always rebase if there are no conflicts (i.e. `git fetch upstream && git rebase upstream/master`).

That's just how I am used to working. Let me know if you prefer working differently and if I should change anything about my "working with git".

**Pull request merged**